settings.py

```diff
@@ -16,16 +16,19 @@ def make_configs(self, build_configs = None):
         run_configs = []
         run_id = 0
         build_config_id = len(build_configs)
-        for run_id, run_config_base in enumerate(self.run_configs_base):
+        for run_config_base in self.run_configs_base:
             run_config = self.process_run_config(run_config_base)
             build_config = self.make_build_config(run_config)
+            if not run_config:
```

Comment:
Shouldn't this be before the build is created? If the run config is skipped, we also do not need the corresponding build config - and typically the make_build_config operates on the run_config and may fail if it is None

Reply:
I do not know why, but if I move the `if` one line higher, the whole script fails...